tests/step-094.sh

```diff
@@ -51,12 +51,12 @@ if ! sh /var/lib/githooks/install.sh; then
 fi
 
 # revert any changes to the cli tool
-cp /tmp/cli-0 "$HOME/.githooks/bin/githooks" &&
-    chmod +x "$HOME/.githooks/bin/githooks" || exit 1
-
-if ! git hooks install; then
-    echo "! The Git alias integration failed: single"
-    # exit 1
+cp /tmp/cli-0 "$HOME/.githooks/release/cli.sh" || exit 1
+OUT=$(git hooks install 2>&1)
+# shellcheck disable=SC2181
+if [ $? -eq 0 ] || ! echo "$OUT" | grep -iq "DEPRECATION WARNING: Single install"; then
```

Comment:
This used to test that the `git hooks install` command works, but now we test that it fails. Perhaps we should unset the single repo is that's why this has changed?